haskell_scotty/ClientRepo.hs

```diff
@@ -29,14 +25,5 @@ saveClient client = do
 deleteClient :: Int -> IO ()
 deleteClient clientId = do
   conn <- connectDB
-  execute conn "DELETE FROM clients WHERE id=?" [clientId]
+  _ <- execute conn "DELETE FROM clients WHERE id=?" [clientId]
```

Comment:
The only other value is that it shows that we are deliberately ignoring a non-unit return result. This is good to know, because it is so often a bad idea. It is arguable that some error-checking code should go here, but I haven't looked into it.